### DISCUSSION TITLE
How to enable Offline Always online and Visible icon?

### DISCUSSION THREAD

**Discussion by breaker84** [LiveHelperChat/livehelperchat#1934] [Q&A]
Dear LCH support.

How do i enable the icon for 3 - Offline, Always online, and Visible icon on top at the name?
I already give permission as guided here - https://doc.livehelperchat.com/docs/invisible-mode/
The icon does not appear in the group operator.

Is there another policy I should give to the role?
Regards

**Comment by remdex** [comment]
All works here. If you are using older version make sure operator relogin
